backend/spec/spec_helper.rb

```diff
@@ -47,7 +47,7 @@
 Capybara.javascript_driver = :poltergeist
 
 # Set timeout to something high enough to allow CI to pass
-Capybara.default_wait_time = 10
+Capybara.default_wait_time = 35
```

Comment:
Sorry I swear my MacBook Air can't finish the suite without raising these. I had meant to *not* check these in, I will revert